remote-application-jasmine.json

```diff
@@ -0,0 +1,26 @@
+{
+  "spec_dir": "test/remote",
```

Comment:
As this file is just for the sake of testing WDIO remote, I think it should be placed under the `test/remote` folder

Reply:
Applied